test/server/config_validation/xds_corpus/example2

```diff
@@ -0,0 +1,32 @@
+actions {
```

Comment:
selfish ask -- can we repro this testcase as a regression testcase?
https://github.com/envoyproxy/envoy/issues/7431

Reply:
what do you mean? add it to `ads_integration_test` or something?

Reply:
Ah, add the sequence of updates as a corpus entry